### ISSUE TITLE
Have a random background image on the landing page, instead of the same all the time.

### ISSUE THREAD

**Issue by victorcorcino** [Terra-Cognita/Terra-Cognita.github.io#1]
Instead of always showing the same background image, we could pick a random image from this folder: https://drive.google.com/drive/folders/19RnWCDsQ7Aeyqr2U3B6PgjChZivbgykT?usp=sharing

**Comment by fhilipk** [comment]
the intention of this is to introduce the player to the port.... perhaps we can re consider the site. I'm not so sure the message of a random image of our concept environments is suppose to convey.

**Comment by victorcorcino** [comment]
Yes, then I believe the intention of the landing page is not clear to all of us. I'd say the intention is to introduce the potential players to the game, not the port necessarily, since the chat bot will answer questions about many other elements of the game. Thoughts?

**Comment by victorcorcino** [comment]
Continuing the discussion here: https://discordapp.com/channels/887412971461025823/955551730844708905/1021094680210968607
